packages/client/lib/execution/vmexecution.ts

```diff
@@ -40,7 +40,7 @@ export class VMExecution extends Execution {
     super(options)
 
     if (isFalsy(this.config.vm)) {
-      const trie = new SecureTrie({ db: new LevelDB(this.stateDB) })
+      const trie = new CheckpointTrie({ db: new LevelDB(this.stateDB), useHashedKeys: true })
```

Comment:
Yeah, really love this new option naming when reading with some distance! 👍 

This is so much easier to grasp than having to deal with what a "Secure Trie" is.